packages/devtools_app/lib/src/ui/flutter/vm_flag_widgets.dart

```diff
@@ -53,10 +64,6 @@ class ProfileGranularityDropdownState
 
   // TODO(kenz): show a warning when ProfileGranularity.high is selected.
   void _onProfileGranularityChanged(String newValue) {
-    if (dropdownValue == newValue) return;
     profilerService.setProfilePeriod(newValue);
```

Comment:
I assume this now works without the setState because a notification gets sent when the value changes. Correct?
The new value shows up almost immediately, right?
I assume the notification from the VM shows up so quickly that the new value updates almost instantly. If not, it could be good to update the new value locally before it shows up from the VM.

Reply:
Correct, the update is almost instant. There is no visual difference from before this change to after.